docs/source/dataframe_schemas.rst

```diff
@@ -703,8 +703,8 @@ data pipeline:
 
     schema = pa.DataFrameSchema(
         columns={
-            "col1": pa.Column(pa.Int, pa.Check(lambda s: s >= 0)),
-            "col2": pa.Column(pa.String, pa.Check(lambda x: x <= 0)),
+            "col1": pa.Column(int, pa.Check(lambda s: s >= 0)),
```

Comment:
The corresponding output in the `.. testoutput::` block also needs to be updated (see line 721)

From the CI error: https://github.com/pandera-dev/pandera/pull/496/checks?check_run_id=2595563461#step:11:330
```
Expected:
    <Schema DataFrameSchema(
        columns={
            'col1': <Schema Column(name=col1, type=int)>
        },
        checks=[],
        coerce=False,
        pandas_dtype=None,
        index=None,
        strict=True
        name=None,
        ordered=False
    )>
Got:
    <Schema DataFrameSchema(
        columns={
            'col1': <Schema Column(name=col1, type=<class 'int'>)>
        },
        checks=[],
        coerce=False,
        pandas_dtype=None,
        index=None,
        strict=True
        name=None,
        ordered=False
    )>
```

There are probably a bunch of other places where this needs to be updated

Reply:
For the CI error I'm not seeing the "Got:" code.  On line 721 it looks to me to be the same as the expected output:

```
    <Schema DataFrameSchema(
        columns={
            'col1': <Schema Column(name=col1, type=int)>
```

Is there a different way I need to format the datatypes in the `testouput::` blocks?

Thanks!

Reply:
yes, so what was `Expected` doesn't match the output that was actually produced, so you'll need to update the `testoutput::` blocks, for example, update this:

```
    <Schema DataFrameSchema(
        columns={
            'col1': <Schema Column(name=col1, type=int)>
        },
        checks=[],
        coerce=False,
        pandas_dtype=None,
        index=None,
        strict=True
        name=None,
        ordered=False
    )>
```

To this:
```
    <Schema DataFrameSchema(
        columns={
            'col1': <Schema Column(name=col1, type=<class 'int'>)>
        },
        checks=[],
        coerce=False,
        pandas_dtype=None,
        index=None,
        strict=True
        name=None,
        ordered=False
    )>
```

Reply:
I made some more updates, I am not sure how to test that these changes worked. Please let me know if things passed. Thanks!